policy/daemon.go

```diff
@@ -100,6 +100,12 @@ func (p *policyd) Start(ctx context.Context) <-chan error {
 		defer close(fch)
 		defer close(ech)
 
+		p.rolePolicies.StartExpired(ctx, p.policyExpiredDuration).
+			EnableExpiredHook().
+			SetExpiredHook(func(ctx context.Context, key string) {
+				//key = <domain>:role.<role>
+				p.fetchAndCachePolicy(ctx, strings.Split(key, ":role.")[0])
+			})
```

Comment:
1. ExpiredHook is called after delete, checking policy between time gap with cause error
2. retry on ExpiredHook fail?
3. fetchAndCachePolicy() error is not print inside the ExpiredHook

Reply:
refer to: https://github.com/yahoojapan/athenz-authorizer/issues/29

Reply:
refer to: https://github.com/yahoojapan/athenz-authorizer/issues/28